content/applications/hr/fleet/accidents.rst

```diff
@@ -0,0 +1,270 @@
+      repairs have not begun. The cost field for this stage remains zero.
+
+   .. tab:: Running
+
+      The repair is in-process but not yet complete. The estimate for repairs is listed in the cost
+      field.
+
+   .. tab:: Completed
+
+      All repairs listed on the service form have been completed. The cost field is updated to
+      reflect the final total cost charged for the repairs.
+
+   .. tab:: Cancelled
+
+      The service request has been canceled. Some common scenarios for this is when the estimate is
+      higher than the remaining vehicle value, and is deemed not worth repairing.
+
+During the repair process, change the service status to reflect the vehicle's current status in one
+of two ways: on the individual :ref:`service record <fleet/service_record>`, or in the :ref:`Kanban
+service view <fleet/Kanban>`.
+
+.. _fleet/service_record:
+
+Service record
+--------------
+
+Open the main :guilabel:`Services` dashboard by navigating to :menuselection:`Fleet app --> Fleet
+--> Services`. Next, click on the individual service record to open the detailed service form. Click
+the desired stage in the top-right corner above the service form to change the status.
+
+.. image:: accidents/running.png
+   :align: center
+   :alt: The stages as seen from the service form.
+
+.. _fleet/Kanban:
+
+Kanban view
+-----------
+
+Open the main :guilabel:`Services` dashboard by navigating to :menuselection:`Fleet app --> Fleet
+--> Services`. First, remove the default :guilabel:`Service Type` filter in the
+:guilabel:`Search...` bar, then click the :icon:`oi-view-kanban` :guilabel:`Kanban` icon in the
+top-right of the screen.
+
+All services appear in a Kanban view, organized by their respective :guilabel:`Status`.
+Drag-and-drop the service record to the desired stage.
+
+.. image:: accidents/drag-n-drop.png
+   :align: center
+   :alt: The Kanban view of stages, with a card being dragged and dropped to the Running stage.
+
+Accident reporting
+==================
+
+One of the main reasons to track accidents using the method outlined in this document is the ability
+to view the total accident cost, determine the safest drivers, and determine the actual total cost
+for specific vehicles.
+
+The main :ref:`Services dashboard <fleet/services_dashboard>` displays all the various accident
+information, while the :ref:`Reporting dashboard <fleet/reporting_dashboard>` displays the total
+cost for specific vehicles.
+
+.. _fleet/services_dashboard:
+
+Services dashboard
+------------------
+
+Navigate to :menuselection:`Fleet app --> Fleet --> Services` to view the default
+:guilabel:`Services` dashboard. All service records are displayed in a :icon:`oi-view-list`
+:guilabel:`(List)` view, grouped alphabetically, by :guilabel:`Service Type`.
+
+The two service types created for accident tracking appear in the list: :guilabel:`Accident -
+Driver Fault` and :guilabel:`Accident - No Fault`.
```

Comment:
I was thinking of the document as a start-to-finish doc, so I assumed that they had created those service types, and were using them. There are only two types of services by default, Leasing and Omnium (no idea  what that is but it's contract related!)

The other types only appear if they are logged as a service. The irony is, you'd never use the Leasing or Omnium ones (since they are labeled as "Contract") so they wouldn't really be a "service". I hope that makes sense.

But yes, if they had configured other services, they'd appear - but organized by type, so you'd still see the 2 they created (in the instructions) and see them at the top (unless they made a Service type that's alphabetically before 'accidents').